src/App.js

```diff
@@ -4,6 +4,7 @@ import { Container, Nav, Navbar } from 'react-bootstrap';
 import HomePage from './home';
 import TransactionPage from './transactions';
 import BudgetPage, { CategoryDetailsPage } from './budget';
+import NewCategoryPage from './budget/NewCategoryPage';
```

Comment:
This import can actually be combined with Line 6. See how we're importing CategoryDetailsPage? We can import NewCategoryPage the same way.